.gitignore

```diff
@@ -69,3 +69,4 @@ testfixtures_shared/
 # Generated
 checkstyle_ide.xml
 x-pack/plugin/esql/src/main/generated-src/generated/
+x-pack/plugin/esql/gen
```

Comment:
This was annoying. It's unrelated.

It's possible my IntelliJ setup needs fixing.